### PR TITLE
ci: added master image publishing

### PR DESCRIPTION
**Pull request by jhellar**
## Motivation

https://issues.jboss.org/browse/AGMS-1135

## Changes

- change version of operand to `master`

### CODE REVIEW

**Review by grdryn — CHANGES_REQUESTED** [review]
The Jenkins pipeline already does the image build & push for the master tag. For now this is the one we prefer, since we trust the infrastructure that this runs on more.

File: Jenkinsfile [hunk 1]
```diff
@@ -158,11 +158,22 @@ pipeline {
...
             }
         }
-        stage("Create a 'latest' tag from 'master'") {
+        stage("Create a 'dev' and 'latest' tag from 'master'") {
             when {
                 branch 'master'
             }
+            environment {
+                // qe-pipeline-library step
+                DOCKER_DEV_TAG = getDevTag()
```
Comment (jhellar) [inline Jenkinsfile:167]: This will be available if [1] is merged.

[1] https://github.com/aerogear/qe-pipeline-library/pull/5

**Comment by jhellar** [comment]
@grdryn ok, I've removed the circleci config. When looking at quay.io and jenkins, seems that latest changes on master were not published. I think that the reason for that is that there was no PR merged for them, the changes were just pushed to master from another branch. Do you know if that is the case? Can we maybe prevent from that by protecting pushing to master?

**Comment by grdryn** [comment]
> @grdryn ok, I've removed the circleci config. When looking at quay.io and jenkins, seems that latest changes on master were not published. I think that the reason for that is that there was no PR merged for them, the changes were just pushed to master from another branch. Do you know if that is the case? Can we maybe prevent from that by protecting pushing to master?

Ugh, that's possible I guess. I've configured the "Merge Button" in a different way than before now, so maybe that will help. I'm also going to look into using prow for these repos, which would help this issue by handling all merges (the merge robot would perform merges rather than us unreliable humans). Until then, if we see that issue again, we can just hit the "build" button in jenkins for the master branch.

File: Jenkinsfile (hunk 1 above)
Comment (jhellar) [reply Jenkinsfile:167]: already merged

**Review by grdryn — APPROVED** [review]
Approved.

Note that I'm still not convinced by this, but let's try it anyway. :+1:

**Pull request merged**